zio-http/src/main/scala/zhttp/http/Response.scala

```diff
@@ -180,9 +180,9 @@ object Response {
   /**
    * Creates a response with content-type set to application/json
    */
-  def json(data: String): Response =
+  def json(data: CharSequence): Response =
     Response(
-      data = HttpData.fromChunk(Chunk.fromArray(data.getBytes(HTTP_CHARSET))),
+      data = HttpData.fromString(data.toString),
```

Comment:
Would be good to push this change into `Http.fromString`, all the way down as low as possible.

Reply:
We can add a new primitive for `HttpData.fromCharSequence` and `Http.fromString` can also use it since `String <: CharSequence`

Reply:
@jdegoes , @tusharmath is there a need for 2 primitives? perhaps just change HttpData.Text to hold a CharSequence

Reply:
I think that's correct, since `CharSequence` is always more efficient than `String`.

Reply:
I pushed change for review . I can add a new method fromCharSequence to replace the fromString method and mark fromString deprecated as it will be subsumed under the method fromCharSequence.

Reply:
I guess the main problem is people will look for `fromString`, expecting it to be there. Not sure.